src/Etterna/Models/StepsAndStyles/Steps.cpp

```diff
@@ -851,10 +856,14 @@ class LunaSteps : public Luna<Steps>
 		auto& ni = nd.SerializeNoteData(etaner);
 		if (ni.size() == 0)
 			return 0;
+		std::vector<float> d;
 
-		std::vector<float> d = MinaSDCalc(ni, rate, goal);
-		auto ssrs = d;
+		if (p->m_StepsType == StepsType_dance_solo) // Single SSR at the moment
```

Comment:
Could you ellaborate on this comment?

Reply:
That probably shouldn't be there, the SoloCalc() call below this line used to only return a float for a single rating, not a vector with different ratings for all skillsets. I ended up just duplicating that one rating for all skillsets because I assumed something would mess up down the line.